test/instances/storage.lua

```diff
@@ -120,6 +120,19 @@ local function bucket_gc_once()
     end)
```

Comment:
typo in commit message: `garbade` -> `garbage`

Reply:
Fixed